test/shared/utils.py

```diff
@@ -85,9 +86,12 @@ def expected_wheels(package_name, package_version):
     else:
         raise Exception('unsupported platform')
     
-    if IS_RUNNING_ON_AZURE:
+    if IS_WINDOWS_RUNNING_ON_AZURE or IS_WINDOWS_RUNNING_ON_TRAVIS:
         # Python 3.4 isn't supported on Azure.
         templates = [t for t in templates if '-cp34-' not in t]
+    if IS_WINDOWS_RUNNING_ON_TRAVIS:
+        # Python 2.7 and 3.4 isn't supported on Travis.
+        templates = [t for t in templates if '-cp27-' not in t and '-cp34-' not in t]
```

Comment:
The 3.4 exclusion is already covered by the previous `if` statement

Reply:
I removed it from previous statement, because whole comment is in this if branch